src/MSBuild.UnitTests/XMake_Tests.cs

```diff
@@ -627,6 +627,7 @@ public void InvalidMaxCPUCountSwitch4()
         [InlineData("-getProperty:Foo;Bar -getItem:MyItem -t:Build", true, "TargetValue", true, true, false, true, false)]
         [InlineData("-getProperty:Foo;Bar -getItem:MyItem", true, "EvalValue", true, false, false, true, false)]
         [InlineData("-getProperty:Foo;Bar -getTargetResult:MyTarget", true, "TargetValue", false, false, true, true, false)]
+        [InlineData("-getProperty:Foo;Bar -getTargetResult:MyTarget -t:restore", true, "TargetValue", false, false, true, true, false)]
```

Comment:
I don't think I understand how this is related to the `-restore -graph` description of the PR.